src/app/pages/ArticlePage/ReadMoreButton/index.styles.ts

```diff
@@ -0,0 +1,58 @@
+import NO_JS_CLASSNAME from '#app/lib/noJs.const';
+import { css, Theme } from '@emotion/react';
+
+export default {
+  readMoreButtonA: ({ spacings, fontSizes, fontVariants }: Theme) =>
+    css({
+      ...fontSizes.pica,
+      ...fontVariants.serifMedium,
+      cursor: 'pointer',
+      fontWeight: 'bold',
+      display: 'block',
+      width: 'calc(100% - 2rem)',
+      padding: `${spacings.DOUBLE}rem 0`,
+      margin: `${spacings.QUADRUPLE}rem ${spacings.DOUBLE}rem -1.5rem  ${spacings.DOUBLE}rem`,
+      backgroundColor: '#141414',
+      color: '#FFFFFF',
+      border: 'none',
+
+      '&:hover, &:focus': {
+        textDecoration: 'underline 2px',
```

Comment:
Can we use relative values please? Also not sure we usually specify the width when it's text underline (just when we use border)